apps/meteor/app/livechat/server/lib/isMessageFromBot.ts

```diff
@@ -0,0 +1,12 @@
+import type { IMessage, IUser } from '@rocket.chat/core-typings';
+import { Users } from '@rocket.chat/models';
+
+export async function isMessageFromBot(message: IMessage): Promise<boolean> {
```

Comment:
Why not use `Users.isUserInRole`?

Reply:
I wasn't aware of this method, definitely worth to use it.

Reply:
Why not use the model method directly instead of creating this new template method (function)?

Reply:
Mainly, for readability.